runtime/plaid/src/loader/mod.rs

```diff
@@ -344,6 +372,50 @@ pub async fn load(
             }
         };
 
+        // Fetch and verify the corresponding signature over this module if we require
+        // rule signing
+        //
+        // If ANY rule does not have a valid signature, we panic instead of continuing because this should be
```

Comment:
Hmm, I'm not sure I agree with this because it means a problem with any rule will prevent Plaid from starting. Is that what we want? Erroring that a particular rule is not sufficiently/correctly signed seems a more sensible approach.

Reply:
This needs updating